vm/kvm-unit-tests/runtest.sh

```diff
@@ -48,198 +54,285 @@ function check_virt_support
     fi
 }
 
-# test is only supported on x86_64, aarch64, ppc64 and s390x
-hwpf=$(uname -i)
-check_platform_support $hwpf
-if (( $? == 0 )); then
-    echo "Running on supported arch ($hwpf)" | tee -a $OUTPUTFILE
+function getTests
```

Comment:
Do we have a naming convention for shell functions? A previous one uses snake_case (`check_platform_support`), it would be nice to be consistent with names.